src/modules/reservation/reservation.service.ts

```diff
@@ -4,47 +4,63 @@ import { omit } from 'radash'
 import { PrismaService } from '@/modules/prisma/prisma.service'
 
 import { ReservationCreateDTO } from './dto/ReservationCreate.dto'
+import { ReservationResponseDTO } from './dto/ReservationResponse.dto'
+import { ReservationResponseFullDTO } from './dto/ReservationResponseFull.dto'
 
 @Injectable()
 export class ReservationService {
     constructor(private prisma: PrismaService) {}
 
-    create(rawData: ReservationCreateDTO) {
+    create(rawData: ReservationCreateDTO): Promise<ReservationResponseFullDTO> {
         return this.prisma.$transaction(async (db) => {
             const ommited = omit(rawData, ['services'])
 
             const newReservation = await db.reservation.create({
                 data: ommited,
+                include: {
+                    services: { include: { service: true } },
+                    HousingUnit: true,
+                },
             })
 
             if (rawData.services) {
-                rawData.services.reservationId = newReservation.id
                 await db.reservationService.createMany({
                     data: rawData.services,
                 })
             }
+
+            return newReservation
         })
     }
 
-    get(id: string) {
-        return this.prisma.reservation.findUnique({ where: { id } })
+    get(id: string): Promise<ReservationResponseFullDTO | null> {
```

Comment:
```suggestion
    getById(id: string): Promise<ReservationResponseFullDTO | null> {
```

Deve ter passado batido em algum PR